examples/custom-snackbar-example-2/src/ReportComplete.tsx

```diff
@@ -74,53 +76,50 @@ const ReportComplete = forwardRef<HTMLDivElement, ReportCompleteProps>(
     }, [id, closeSnackbar]);
 
     return (
-      <SnackbarContent ref={ref} className={classes.root}>
-        <Card className={classes.card} style={{ backgroundColor: "#fddc6c" }}>
-          <CardActions classes={{ root: classes.actionRoot }}>
-            <Typography variant="body2" className={classes.typography}>
-              {props.message}
-            </Typography>
-            <div className={classes.icons}>
-              <IconButton
+      <StyledSnackbarContent ref={ref}>
+        <StyledCard>
+          <StyledCardActions>
+            <StyledTypography variant="body2">
+              {message}
+            </StyledTypography>
+            <div>
+              <StyledIconButton
                 aria-label="Show more"
                 size="small"
-                className={clsx(classes.expand, {
-                  [classes.expandOpen]: expanded
-                })}
                 onClick={handleExpandClick}
+                className={expanded ? 'expandOpen' : ''}
               >
                 <ExpandMoreIcon />
-              </IconButton>
-              <IconButton
+              </StyledIconButton>
+              <StyledIconButton
                 size="small"
-                className={classes.expand}
                 onClick={handleDismiss}
               >
                 <CloseIcon fontSize="small" />
-              </IconButton>
+              </StyledIconButton>
             </div>
-          </CardActions>
+          </StyledCardActions>
           <Collapse in={expanded} timeout="auto" unmountOnExit>
-            <Paper className={classes.paper}>
+            <StyledPaper>
```

Comment:
```suggestion
            <Paper sx={{ bgColor: '#fff', p: 2 }}>
```
I think it is better to use sx prop instead of a styled component so that it can be easier to sync with the theme. ( I think even the background color should some from the theme as well)